src/_testing_/operate.test.js

```diff
@@ -0,0 +1,27 @@
+/*eslint-disable*/
+import operate from '../logic/operate';
+
+describe('check all Arithmetic operations', () => {
+  test('performs addition correctly', () => {
+    expect(+operate(3, 5, '+')).toBe(8);
+  });
+
+  test('Subtract numbers corerctly', () => {
+    expect(+operate(3, 5, '-')).toBe(-2);
+  });
+
+  test('Multiply numbers correctly', () => {
+    expect(+operate(2, 2, 'x')).toBe(4);
+  });
+
+  test('Divide numbers correctly', () => {
+    expect(+operate(10, 2, '÷')).toBe(5);
+  });
+  test('Handle dividing by 0', () => {
+    expect(operate(10, 0, '÷')).toBe("Can't divide by 0.");
+  });
+
```

Comment:
- [ ] It will be great to add a test for the modulo operation (%). It seems you missed to add the test for this operation.